src/actions/config.js

```diff
@@ -82,6 +83,25 @@ export function configDidLoad(config) {
   };
 }
 
+const URL_BASE = 'https://raw.githubusercontent.com/netlify/netlify-cms/master/src/'
```

Comment:
Why you all insist on i18n string distribution via patches? The end user would need rebundle in all the cases you have described. In contrast to my approach that loads the strings on app boostrap. I can see only pros.